No plugin found for a rule: 'sort-imports-by-specifiers'

I haven't run `putout` in a while in my repo, but have reinstalled deps several times since last time, and now when I'm running `putout` I get this error message per file:

```
No plugin found for a rule: 'sort-imports-by-specifiers'
```

I see that no plugin has been installed for that rule, and I see in another older repo, I have that plugin. What gives? 

I even tried to `npm i -D putout@latest` (giving `^38.1.2`) to see if I got the plugin back, but with no help. I also have no CLI memory (`ctrl+r`) of ever installing plugins manually.

```
$ node -v && npm -v
v22.14.0
11.1.0
```

```
$ npx putout --fix .putout.json
.putout.json
 1:1  error   No plugin found for a rule: 'sort-imports-by-specifiers'  loader 

✖ 1 errors in 1 files
  fixable with the `--fix` option
```

The error is still there, but I see that it should be `esm/sort-imports-by-specifiers`?

--- a/packages/plugin-putout-config/lib/apply-esm/fixture/apply-esm-fix.js
+++ b/packages/plugin-putout-config/lib/apply-esm/fixture/apply-esm-fix.js
@@ -1,11 +1,13 @@
+/* eslint-disable */
 __putout_processor_json({
-    "rules": {
+    rules: {
         "esm/convert-assert-to-with": "off",
         "esm/remove-empty-import": "on",
         "esm/remove-empty-export": "on",
         "esm/group-imports-by-source": "on",
         "esm/declare-imports-first": "on",
         "esm/remove-quotes-from-import-assertions": "on",
+        "esm/sort-imports-by-specifiers": "on",
         "esm/merge-duplicate-imports": "on"
     }
 });

--- a/packages/plugin-putout-config/lib/apply-esm/fixture/apply-esm.js
+++ b/packages/plugin-putout-config/lib/apply-esm/fixture/apply-esm.js
@@ -1,11 +1,13 @@
+/* eslint-disable */
 __putout_processor_json({
-    "rules": {
+    rules: {
         "convert-assert-to-with": "off",
         "remove-empty/import": "on",
         "remove-empty/export": "on",
         "group-imports-by-source": "on",
         "declare-imports-first": "on",
         "remove-quotes-from-import-assertions": "on",
+        "sort-imports-by-specifiers": "on",
         "merge-duplicate-imports": "on"
-  }
+    }
 });

--- a/packages/plugin-putout-config/lib/apply-esm/index.js
+++ b/packages/plugin-putout-config/lib/apply-esm/index.js
@@ -9,6 +9,7 @@ const v37 = [
     ['group-imports-by-source', 'esm/group-imports-by-source'],
     ['declare-imports-first', 'esm/declare-imports-first'],
     ['remove-quotes-from-import-assertions', 'esm/remove-quotes-from-import-assertions'],
+    ['sort-imports-by-specifiers', 'esm/sort-imports-by-specifiers'],
     ['merge-duplicate-imports', 'esm/merge-duplicate-imports'],
 ];
 

--- a/packages/plugin-putout-config/test/fixture/apply-esm-fix.js
+++ b/packages/plugin-putout-config/test/fixture/apply-esm-fix.js
@@ -1,10 +1,12 @@
+/* eslint-disable */
 __putout_processor_json({
-    "rules": {
+    rules: {
         "esm/remove-empty-import": "on",
         "esm/remove-empty-export": "on",
         "esm/group-imports-by-source": "on",
         "esm/declare-imports-first": "on",
         "esm/remove-quotes-from-import-assertions": "on",
+        "esm/sort-imports-by-specifiers": "on",
         "esm/merge-duplicate-imports": "on"
     }
 });

--- a/packages/plugin-putout-config/test/fixture/apply-esm.js
+++ b/packages/plugin-putout-config/test/fixture/apply-esm.js
@@ -1,10 +1,12 @@
+/* eslint-disable */
 __putout_processor_json({
-    "rules": {
+    rules: {
         "remove-empty/import": "on",
         "remove-empty/export": "on",
         "group-imports-by-source": "on",
         "declare-imports-first": "on",
         "remove-quotes-from-import-assertions": "on",
+        "sort-imports-by-specifiers": "on",
         "merge-duplicate-imports": "on"
-  }
+    }
 });

